### DISCUSSION TITLE
Ggrep without opening the first result

### DISCUSSION THREAD

**Discussion by byron-hawkins** [tpope/vim-fugitive#1731] [Q&A] [answered]
When I search with `Ggrep`, it populates the `quickfix` window, and at the same time opens the first matching file and navigates to the first matching line in that file. Is it possible for `Ggrep` to populate the `quickfix` window without opening any file or navigating anywhere? I just want to grep and see the list of results. (Note that automatically opening `quickfix` after `Ggrep` is easy).

**Comment by tpope** [comment, marked as answer]
`:Ggrep!`, same as `:grep!`.

**Reply by byron-hawkins** [reply]
Great. I was searching for something like `:grep!` but after 10 minutes could not find anything, and reading `help grep` was not leading to that idea. Thanks.

**Reply by tpope** [reply]
It's a bit obtuse, `:help :grep` references `:help :make` which fully explains it.
